src/lib/hooks/prerun.ts

```diff
@@ -13,10 +13,8 @@ const hook: Hook.Prerun = async function run(options) {
   if (options.Command?.flags?.key) {
     this.debug('current command has --key flag');
     if (isTest()) {
-      // eslint-disable-next-line no-param-reassign
```

Comment:
Not having a `no-param-reassign` rule might be an issue but with TS we've grown to be pretty good at not writing code that gets flagged with it.